http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleDataprocDAO.scala

```diff
@@ -132,19 +131,6 @@ class HttpGoogleDataprocDAO(
     transformed.value.handleGoogleException(googleProject, clusterName)
   }
 
-  override def getClusterInstances(googleProject: GoogleProject,
```

Comment:
Assuming this moved to wb-libs

Reply:
yea https://github.com/broadinstitute/workbench-libs/pull/300/files#diff-28a9648a50715239edaa8ca1bfc1a263R130